services/message-bus/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const createRequestService = require('../request');
+
+function createMessageBusCaller(opts) {
```

Comment:
I wonder if our factory functions should use "make" instead of "create". Might avoid confusion with creating db stuff, just a thought!